src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/task/respondtodefencespeccallbackhandlertask/DetermineNextState.java

```diff
@@ -118,7 +118,8 @@ private Pair<String, BusinessProcess> handleAcceptedRepaymentPlan(CaseData caseD
                                                BusinessProcess businessProcess) {
         String nextState;
         if (featureToggleService.isJudgmentOnlineLive()
-            && (caseData.isPayByInstallment() || caseData.isPayBySetDate())) {
+            && (caseData.isPayByInstallment() || caseData.isPayBySetDate())
+            && caseData.isLRvLipOneVOne()) {
```

Comment:
Adding this condition , might change how the submission page changes for LIP vs LIP and LR/LIP vs LR.